zebra-rpc/qa/rpc-tests/wallet.py

```diff
@@ -48,11 +48,6 @@ def run_test(self):
             print("Ignoring stopping wallet error: ", e)
         time.sleep(1)
 
-        # Hack for https://github.com/ZcashFoundation/zebra/issues/9708
-        # We Stop the wallet which has 1 block, generate 100 blocks in zebra,
-        # so when restarting Zebra it will have 1 block, just as the wallet.
-        self.nodes[0].generate(100)
```

Comment:
This should wait 6 seconds for Zebra to write to the non-finalized state to the backup cache (updates to the backup are rate-limited to once per 5 seconds, plus 1 second for I/O).

Reply:
You are right, 6 seconds is safer. Changed at https://github.com/ZcashFoundation/zebra/pull/9896/commits/9ee495821681c8b7e0dd1c1f365b3b25ef00677b